nomos-services/membership/src/lib.rs

```diff
@@ -129,6 +131,12 @@ where
             <RuntimeServiceId as AsServiceId<Self>>::SERVICE_ID
         );
 
+        wait_until_services_are_ready!(
+            &self.service_resources_handle.overwatch_handle,
+            Some(Duration::from_millis(3000)),
+            <S as SdpAdapter>::SdpService
+        );
```

Comment:
Are you gonna expand later for all services in this PR or different one?

Reply:
Different. I added that just as PoC (albeit maybe I shouldn't have). I mention that in the PR description.